Prolangle/Languages/Framework/SyntaxStyle.cs

```diff
@@ -8,6 +8,9 @@ public enum SyntaxStyle
 	None,
 	Other,
 
+	[Description("In the assembly syntax style, you use mnemonics representing" +
+	             "machine instructions and can pass them arguments in the " +
```

Comment:
I think there needs to be a space after `representing` or before `machine`?

Reply:
Fixed